ods_ci/tests/Tests/400__ods_dashboard/420__model_serving/LLMs/vllm/426__model_serving_vllm_metrics.robot

```diff
@@ -89,7 +103,7 @@ Verify Vllm Metrics Are Present
     Set Suite Variable    ${token}
     Metrics Should Exist In UserWorkloadMonitoring    ${thanos_url}    ${token}    ${SEARCH_METRICS}
 
-Verify Vllm Metrics Values Match Between UWM And Endpoint
+Verify Vllm And tgi Metrics Values Match Between UWM And Endpoint
```

Comment:
This does not work - Line 98/112 fetches the metrics to test for by looking at those that start with `vllm:`, so TGI metrics are not being tested at all here.
You will need to either change the logic of the `Get Vllm Metrics And Values` in Helpers.py or develop an equivalent keyword for TGI metrics